code/modules/clothing/rogueclothes/gloves/gloves.dm

```diff
@@ -203,11 +203,11 @@
 	item_state = "psydongloveschains"
 
 /obj/item/clothing/gloves/roguetown/otavan
-	name = "otavan leather gloves"
-	desc = "A pair of heavy Otavan leather gloves, commonly used by fencers, renowned for their quality."
+	name = "fencer leather gloves"
+	desc = "A pair of heavy leather gloves, commonly used by fencers, renowned for their quality."
 	icon_state = "fencergloves"
 	item_state = "fencergloves"
-	armor = list("blunt" = 60, "slash" = 100, "stab" = 80, "piercing" = 20, "fire" = 0, "acid" = 0)
+	armor = list("blunt" = 60, "slash" = 100, "stab" = 80, "bullet" = 20, "fire" = 0, "acid" = 0)
```

Comment:
If their shtick is good for fencing, it makes sence for them to have high stab protection.
That said, Their blunt and slash protection could use a solid nerf.